java/arcs/android/sdk/host/IntentArcHostAdapter.kt

```diff
@@ -98,12 +99,14 @@ class IntentArcHostAdapter(
     private suspend fun <T> sendIntentToArcHostServiceForResult(
         intent: Intent,
         transformer: (Any?) -> T?
-    ): T? = suspendCancellableCoroutine { cancelableContinuation ->
-        ArcHostHelper.setResultReceiver(
-            intent,
-            ResultReceiverContinuation(cancelableContinuation, transformer)
-        )
-        sendIntentToArcHostService(intent)
+    ): T? = withTimeout(2000L) {
```

Comment:
How did you arrive at this timeout value? Would it make sense to define a symbol for this?

Reply:
I added a symbol. This was added because unit test fails would deadlock forever on GCB and slow down the build. Additionally, I noticed 1s lead to flaky GCB behavior (doesnt seem to be needed on local mac), so experimentally 2s seemed best.